holoviews/core/data/__init__.py

```diff
@@ -516,13 +516,17 @@ def get_dimension_type(self, dim):
         return self.interface.dimension_type(self, dim_obj)
 
 
-    def dframe(self, dimensions=None):
+    def dframe(self, dimensions=None, copy=True):
```

Comment:
That's true, it's more like ``avoid_copy``, but I think providing a consistent API to get a hold of a dataframe with the minimal amount of overhead is useful.